docs/config-schema.md

```diff
@@ -56,6 +56,7 @@ if not set.
 | `metricsToInclude` | no | [list of objects (see below)](#metricstoinclude) | A list of metric filters that will whitelist/include metrics.  These filters take priority over the filters specified in `metricsToExclude`. |
 | `metricsToExclude` | no | [list of objects (see below)](#metricstoexclude) | A list of metric filters |
 | `propertiesToExclude` | no | [list of objects (see below)](#propertiestoexclude) | A list of properties filters |
+| `enableExtraMetricsFilter` | no | bool | Whether to enable new-style metrics filter (**default:** `false`) |
```

Comment:
Does it break the tests horribly if it already defaults to `true`?

Reply:
Sorry, this would never default to true since existing configs won't have it.  I should have said can you go ahead and set it to true in the package configs we have scattered throughout this repo?